src/main/scala/no/ndla/imageapi/controller/RawController.scala

```diff
@@ -75,11 +75,10 @@ trait RawController {
     }
 
     private def getRawImage(imageName: String): ImageStream = {
-      val dynamicCropOrResize = if (canDoDynamicCrop) dynamicCrop _ else resize _
-
       imageStorage.get(imageName)
         .flatMap(crop)
-        .flatMap(dynamicCropOrResize) match {
+        .flatMap(dynamicCrop)
+        .flatMap(resize) match {
```

Comment:
`dynamicCrop` considers focal points (if present), and must therefore be performed _before_ we (potentially) scale the image.